mws/resource_databricks_workspace_conf.go

```diff
@@ -0,0 +1,90 @@
+package mws
+
+// Preview feature: https://docs.databricks.com/security/network/ip-access-list.html
+// REST API: https://docs.databricks.com/dev-tools/api/latest/ip-access-list.html#operation/create-list
+
```

Comment:
The workspace_conf resource objects should go into the workspace package rather than mws. This api will be used beyond just mws.